tests/cluster/tests/20-half-migrated-slot.tcl

```diff
@@ -32,55 +32,58 @@ reset_cluster
 $cluster set aga xyz
 
 test "Half init migration in 'migrating' is fixable" {
-    $nodefrom(link) cluster setslot 609 migrating $nodeto(id)
+    assert_equal {OK} [$nodefrom(link) cluster setslot 609 migrating $nodeto(id)]
     fix_cluster $nodefrom(addr)
-    assert {[$cluster get aga] eq "xyz"}
+    assert_equal "xyz" [$cluster get aga]
 }
 
 test "Half init migration in 'importing' is fixable" {
-    $nodeto(link) cluster setslot 609 importing $nodefrom(id)
+    assert_equal {OK} [$nodeto(link) cluster setslot 609 importing $nodefrom(id)]
     fix_cluster $nodefrom(addr)
-    assert {[$cluster get aga] eq "xyz"}
+    assert_equal "xyz" [$cluster get aga]
 }
 
 test "Init migration and move key" {
-    $nodefrom(link) cluster setslot 609 migrating $nodeto(id)
-    $nodeto(link) cluster setslot 609 importing $nodefrom(id)
-    $nodefrom(link) migrate $nodeto(host) $nodeto(port) aga 0 10000
-    assert {[$cluster get aga] eq "xyz"}
+    assert_equal {OK} [$nodefrom(link) cluster setslot 609 migrating $nodeto(id)]
+    assert_equal {OK} [$nodeto(link) cluster setslot 609 importing $nodefrom(id)]
+    assert_equal {OK} [$nodefrom(link) migrate $nodeto(host) $nodeto(port) aga 0 10000]
+    wait_for_cluster_propagation
+    assert_equal "xyz" [$cluster get aga]
     fix_cluster $nodefrom(addr)
```

Comment:
WTF????? Is it test or piece of sh#t? Why the hell we wait for cluster propagation before we call fix_cluster????????